Formula/owamp.rb

```diff
@@ -17,6 +17,8 @@ class Owamp < Formula
   # Fix to prevent tests hanging under certain circumstances.
   # Provided by Aaron Brown via perfsonar-user mailing list:
   # https://lists.internet2.edu/sympa/arc/perfsonar-user/2014-11/msg00131.html
```

Comment:
No, it's not in the master branch of perfsonar/owamp. I will submit upstream.

(Thanks for all other comments - I won't have time to re-integrate for another week or two, but I will do so)